### DISCUSSION TITLE
Megadepth image can not be read

### DISCUSSION THREAD

**Discussion by HUSTNO1WXY** [zju3dv/LoFTR#124] [Q&A] [answered]
![T~YP8{E3R}7 B$0R%2C HFE](https://user-images.githubusercontent.com/41225553/145699237-590a6b8b-3359-429b-84fd-59ccb53d93ff.png)
I have build meagdepth data in "data/train" and "data/test"， but when I train, I found it can't be read. Why does this bug happen? The paths I have bulit are "D:\LoFTR-master\data\megadepth\train\phoenix\S6\zl548\MegaDepth_v1" and "D:\LoFTR-master\data\megadepth\train\Undistorted_SfM"

**Comment by HUSTNO1WXY** [comment, marked as answer]
Path in Scannet is `img_name0 = osp.join(self.root_dir, scene_name, 'color', f'{stem_name_0}.jpg')`, while path in  Megadepth is `img_name0 = osp.join(self.root_dir, self.scene_info['image_paths'][idx0])`. So, the second code will cause path in Megadepth become ''data/megadepth/test\\Undistorted_SfM/0022/images/2541218468_a70d8e4860_o.jpg", which is different from Scannet path 'data/scannet/test\\scene0707_00\\color\\45.jpg'

**Reply by HUSTNO1WXY** [reply]
> 

And the code will not read path in Megadepth.

**Reply by HUSTNO1WXY** [reply]
And I want to know how to solve this problem

**Reply by HUSTNO1WXY** [reply]
Oh, I‘m sorry. I put file in wrong place

**Reply by zhubingerdai** [reply]
I would like to ask whether the path of default.py in the config file needs to be changed after the address is correctly placed. I don't know which ones to modify, but I can't read them now either。Is your dataset path set up and ready to run？？
